docs/source/reference/glossary.rst

```diff
@@ -1,6 +1,72 @@
 Glossary
 ========
 
+
+API
+---
+
+application programming interface
+
+CLI
+---
+
+command line interface
```

Comment:
Can we add to this: "The CLI is typically how we interact with MatFlow on HPC systems."